Cargo.toml

```diff
@@ -4,4 +4,4 @@
 h2 = { git = "https://github.com/kate-goldenring/h2", branch = "master"}
 
 [workspace]
-members = ["shared", "controller", "agent", "samples/brokers/udev-video-broker"]
\ No newline at end of file
+members = ["shared", "controller", "agent", "samples/brokers/udev-video-broker", "samples/brokers/nessie" ]
```

Comment:
Do you think it would be helpful to include actual Nessie code?  We have resisted and kept it as a doc sample up to now, but it would be interesting to hear the other side of that.

Reply:
If we aren't including the Nessie source code, this probably shouldn't be included.

Reply:
One advantage in having the example as code is that you can run it through the CI tool and ensure it builds.

The disadvantage is in needing to chop it up to form the documentation.

There must (!?) be something akin to a Jupyter notebook that can comprise build/run-able code and Markdown as comments that can be rendered on-the-fly as a tutorial.

In this case, the documentation omits this change and, as a result, `nessie.rs` does not get auto-generated from the proto.

Reply:
Totally agree with everything you said.  100%.

If we are going to keep this as an md-based how-to, we need to find a way to validate the code changes (and keep them in sync).

Reply:
Validating the changes easily would be a really nice to have. 
Since this addition is needed to auto-generate the code, @DazWilkin, could you add a step to the extensibility doc to add nessie to the `Cargo.toml`?

Reply:
@kate-goldenring I realized that your `Dockerfile` adds `./samples/brokers/nessie` to `members` for the build to succeed.

So, the developer could either add the project to `Cargo.toml`?

Or they could `cd ./samples/brokers/nessie && cargo build` to generate it (while developing).

Do you have a preference? I'll then add.

Reply:
Since anyone doing this will have to build a new agent as well, it seems like adding nessie to the /Cargo.toml's member list makes the most sense.